app/page/template/_dist/app.htm

```diff
@@ -178,6 +178,7 @@
 <script src="/script/cryptography/CryptographyMapper.js"></script>
 <script src="/script/cryptography/CryptographyRepository.js"></script>
 <script src="/script/cryptography/CryptographyService.js"></script>
+<script src="/script/cryptography/SessionID.js"></script>
```

Comment:
`SessionID.js` sounds like it is an entity. I would prefer a `SessionHelper` or `SessionUtil`.